exporters/requirements.txt

```diff
@@ -1,10 +1,19 @@
-requests
-prometheus_client
-openshift < 0.12
-jsonpath_ng
-jira
-pytz
-python-gitlab >= 2.4.0
-git-url-parse
-azure-devops
-attrs
+# Common
+openshift < 0.12       # module openshift
+prometheus_client      # module prometheus_client
+
+# Committime exporter
```

Comment:
Mind if I add notes to here about which backends these are used for and/or why they're used?

Or is this generated by one of the tools you mentioned, so we shouldn't touch it?

Reply:
Added as it's used by all of them.